benchmarks/expected-plans/q4.txt

```diff
@@ -5,13 +5,14 @@
 |               |   Projection: orders.o_orderpriority, COUNT(UInt8(1)) AS order_count                                                                                          |
```

Comment:
I reviewed these plans and they see to be all the same, structurally (there are some aliases that are different)